conda.recipe/meta.yaml

```diff
@@ -1,10 +1,13 @@
+{% set data = load_setup_py_data() %}
+
 package:
   name: bokeh
-  version: 'nightly'
+  version: {{ data['version'] }}
+
+source:
+  path: ..
```

Comment:
use of `__conda_version__.txt` was removed in this PR. I'm on the fence about the `path`. There are advantages and disadvantages both ways. `..` seems best for now